packages/location-field/src/index.js

```diff
@@ -326,11 +341,19 @@ class LocationField extends Component {
       operatorIconMap
     } = this.props;
     const { activeIndex } = this.state;
+
+    // generate the friendly labels for this feature
+    const { main, secondary } = generateLabel(feature.properties);
+
     // Create the selection handler
     const locationSelected = () => {
       getGeocoder(geocoderConfig)
         .getLocationFromGeocodedFeature(feature)
         .then(geocodedLocation => {
+          // add the friendly location labels
+          // FIXME: should this be a copy of the object so it's not mutated?
+          geocodedLocation.main = main;
+          geocodedLocation.secondary = secondary;
           // Set the current location
           this.setLocation(geocodedLocation, "GEOCODE");
```

Comment:
we may want to consider overwriting `name`. I don't see us needing the original name ever, and not setting it causes otp-rr to put the wrong string in the URL path. 

To avoid a breaking change we could add a new prop for `nameOverride` and then create a second PR in otp-rr that uses that new prop? The other alternative is to patch OTP-RR to assemble the query parameter differently, which will be a lot more work than this fix.

Reply:
I started looking into OTP-RR and I think making changes there would be a biiig project. It involves not just history, but also saved trips, and probably other stuff. I decided to replace the name field and it seems to work fine. Old saved trips will just have the "old" format, which doesn't strike me as a "breaking" change.